### ISSUE TITLE
Get_domain failed

### ISSUE THREAD

**Issue by AngryOnigiri** [mitre/caldera#340]
Hello, 

I've been using the old caldera version 0.3.0 have the following issue:

```
WARNING:app.operation.operation:Failed to parse with error: 'Result is not well formed:     
Ethernet:
Knoten-IP-Adresse: [10.0.2.9] Bereichskennung: []

                Lokale NetBIOS-Namentabelle

       Name               Typ          Status
    ---------------------------------------------
    DOMAINUSER1    <00>  EINDEUTIG   Registriert 
    NEWDOMAIN      <00>  GRUPPE      Registriert 
    DOMAINUSER1    <20>  EINDEUTIG   Registriert 
'
INFO:app.operation.operation:Returned from get_domain

```

![caldera](https://user-images.githubusercontent.com/50779286/62220543-7e887980-b3b0-11e9-9bab-47f5deb03164.PNG)

Is it because my system language is in german? 

The RAT is running on a win10 domainuser (domain admin).
I've started the operation with an existing RAT and also bootstrap RAT as active user (domain admin).

**Comment by ArtificialErmine** [comment]
Yeah, this looks like it's an issue with language settings. Although this is an outdated version of Caldera, and we don't currently have any plans to upgrade Adversary mode (the modern incarnation of this system) to support multiple languages, you should be able to fix the problem yourself by modifying `regex = re.compile(r'\s*(\S+)\s*<[0-9][0-9]>\s*GROUP')` to something like `regex = re.compile(r'\s*(\S+)\s*<[0-9][0-9]>\s*GRUPPE')` in the `n` method within the `nbstat` class object in `app\commands\parsers`. @AngryOnigiri Let me know whether or not that fixes that issue.
